src/components/label/label.tsx

```diff
@@ -26,7 +27,17 @@ export const Label: FC<Props> = ({ label, description, isOptional = false }) =>
           </>
         )}
       </View>
-      {isDefined(description) && <Text style={styles.description}>{description}</Text>}
+      {isDefined(description) && (
+        <Text style={styles.description}>
+          {Array.isArray(description)
+            ? description.map((item, index) => (
+                <Text key={index} style={item.bold === true ? styles.boldDescriptionPiece : undefined}>
+                  {item.text}
+                </Text>
+              ))
+            : description}
+        </Text>
+      )}
```

Comment:
Can we create a new component, called `Description` and move this logic into it?